packages/forms-web-app/src/routes/register/agent/what-their-address.js

```diff
@@ -1,21 +1,20 @@
 const express = require('express');
 
-const addressController = require('../../../controllers/register/behalf/representee-address');
+const addressController = require('../../../controllers/register/agent/what-their-address');
 const { validationErrorHandler } = require('../../../validators/validation-error-handler');
 const {
   rules: addressValidationRules,
-} = require('../../../validators/register/behalf/representee-address');
-
+} = require('../../../validators/register/agent/what-their-address');
 
 const router = express.Router();
 
-router.get('/representee-address', addressController.getAddress);
+router.get('/what-their-address', addressController.getAddress);
 
 router.post(
-  '/representee-address',
+  '/what-their-address',
```

Comment:
I appreciate you've implemented what's on the story but I don't like `/register/agent/what-their-address` at all.  I'll raise with Aissah and Phil.

Reply:
So agreement to use `/register/agent/their-postal-address` - I have updated the story.  Please could you revise @lvnhmd ?